docs/management_api.yml

```diff
@@ -571,9 +571,11 @@ paths:
           in: query
```

Comment:
maybe we can completely drop the parameter from doc

Reply:
I think it'd be ok to remove it. Not an API expert though :)

Reply:
Removed from API docs. Simplified test cases to only verify that expire is ignored.